### DISCUSSION TITLE
Task AMDRyzenMasterSDKTask : Installed but SDK no.

### DISCUSSION THREAD

**Discussion by kalimaenergy** [GSDragoon/RadeonSoftwareSlimmer#23] [Ideas]
Hello,

Maybe if you can add on task list "AMDRyzenMasterSDKTask" for force disabled.

I unselect AMDRyzenMasterSDK of components, but the task is still here. 

Thank you and have a nice day,

**Comment by GSDragoon** [comment]
If you disable the task, it will still be installed, but disabled (vs removing it). If it's being disabled before install and somehow being re-enabled, then the installer might be forcing enabling it. Can you disable it under Post Install? If not, can you export the scheduled task (Task Scheduler -> right click on the scheduled task -> select Export) and attach the xml file here?

**Reply by kalimaenergy** [reply]
Sorry for my english, it's not my native language:

- I disabled RyzenmasterSDK components on pre-install
- I make a complete DDU and install light driver
- RyzenMasterSDK not installed but Task installed and enabled (not run) 
- I don't make any things with post install.
